### ISSUE TITLE
App: Error in "Open Packaging Format"

### ISSUE THREAD

**Issue by davispuh** [jellyfin/jellyfin-plugin-bookshelf#14]
Log spammed with
```
[2020-06-13 00:54:15.085 +03:00] [ERR] [71] App: Error in "Open Packaging Format"
System.NullReferenceException: Object reference not set to an instance of an object.
   at Jellyfin.Plugin.Bookshelf.Providers.BookProviderFromOpf.ReadOpfData(MetadataResult`1 bookResult, String metaFile, CancellationToken cancellationToken)
   at Jellyfin.Plugin.Bookshelf.Providers.BookProviderFromOpf.GetMetadata(ItemInfo info, IDirectoryService directoryService, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.MetadataService`2.RefreshWithProviders(MetadataResult`1 metadata, TIdType id, MetadataRefreshOptions options, List`1 providers, ItemImageProvider imageService, CancellationToken cancellationToken) in /mnt/AUR/jellyfin/src/jellyfin-10.5.5/MediaBrowser.Providers/Manager/MetadataService.cs:line 708
[2020-06-13 00:54:16.129 +03:00] [ERR] [50] App: Error in "Open Packaging Format"
System.NullReferenceException: Object reference not set to an instance of an object.
   at Jellyfin.Plugin.Bookshelf.Providers.BookProviderFromOpf.ReadOpfData(MetadataResult`1 bookResult, String metaFile, CancellationToken cancellationToken)
   at Jellyfin.Plugin.Bookshelf.Providers.BookProviderFromOpf.GetMetadata(ItemInfo info, IDirectoryService directoryService, CancellationToken cancellationToken)
   at MediaBrowser.Providers.Manager.MetadataService`2.RefreshWithProviders(MetadataResult`1 metadata, TIdType id, MetadataRefreshOptions options, List`1 providers, ItemImageProvider imageService, CancellationToken cancellationToken) in /mnt/AUR/jellyfin/src/jellyfin-10.5.5/MediaBrowser.Providers/Manager/MetadataService.cs:line 708
```

**System:**
 - OS: Arch Linux
 - Virtualization: None
 - Clients: Browser
 - Browser: Firefox
 - Jellyfin Version: 10.5.5
 - Playback: Transcode
 - Installed Plugins: AudioDB, Bookshelf, Fanart, LDAP-Auth, MusicBrainz, Open Subtitles, Playback Reporting, Reports
 - Reverse Proxy: nginx
 - Base URL: none
 - Networking: NAT
 - Storage: local

**Comment by anthonylavado** [comment]
Moved to the bookshelf plugin repository. As per #13, which version of the plugin are you using?

**Comment by itegulov** [comment]
I have also experienced this and it should be fixed by https://github.com/jellyfin/jellyfin-plugin-bookshelf/pull/10. In particular, https://github.com/jellyfin/jellyfin-plugin-bookshelf/pull/10/commits/55a0928a5fee5f25a1c5ccafa716894054c975c6 introduces additional null checks to OPF reader.

**Comment by davispuh** [comment]
That's nice, but that branch haven't been merged. I'm using newest version that there is in list `3.0.0.0`

**Comment by oddstr13** [comment]
Merged and in plugin repos for Jellyfin 10.6.0
